dcm4che-net/src/main/java/org/dcm4che3/net/Device.java

```diff
@@ -1164,6 +1171,27 @@ public final TrustManager getTrustManager() {
         return tm;
     }
 
+    private final ApplicationEntity findApplicationEntityByAeTitleOrAlias(String aet) {
```

Comment:
Not sure if this method could be used by multiple threads concurrently?
Then you would have to think a bit more about correct synchronization.